docs/src/modules/components/MaterialUIComponents/MaterialInputComponents.js

```diff
@@ -0,0 +1,157 @@
+    srcLight: '/static/material-ui/react-components/radio-group-light.png',
+    srcDark: '/static/material-ui/react-components/radio-group-dark.png',
+    link: '/material-ui/react-radio-group/',
+    md1: false,
+    md2: true,
+    md3: false,
+    noGuidelines: false,
+  },
+  {
+    name: 'Rating',
+    srcLight: '/static/material-ui/react-components/rating-light.png',
+    srcDark: '/static/material-ui/react-components/rating-dark.png',
+    link: '/material-ui/react-rating/',
+    md1: false,
+    md2: false,
+    md3: false,
+    noGuidelines: true,
+  },
+  {
+    name: 'Select',
+    srcLight: '/static/material-ui/react-components/select-light.png',
+    srcDark: '/static/material-ui/react-components/select-dark.png',
+    link: '/material-ui/react-select/',
+    md1: false,
+    md2: true,
+    md3: false,
+    noGuidelines: false,
+  },
+  {
+    name: 'Slider',
+    srcLight: '/static/material-ui/react-components/slider-light.png',
+    srcDark: '/static/material-ui/react-components/slider-dark.png',
+    link: '/material-ui/react-slider/',
+    md1: false,
+    md2: true,
+    md3: true,
+    noGuidelines: false,
+  },
+  {
+    name: 'Switch',
+    srcLight: '/static/material-ui/react-components/switch-light.png',
+    srcDark: '/static/material-ui/react-components/switch-dark.png',
+    link: '/material-ui/react-switch/',
+    md1: false,
+    md2: true,
+    md3: false,
+    noGuidelines: false,
+  },
+  {
+    name: 'Text Field',
+    srcLight: '/static/material-ui/react-components/text-field-light.png',
+    srcDark: '/static/material-ui/react-components/text-field-dark.png',
+    link: '/material-ui/react-text-field/',
+    md1: false,
+    md2: true,
+    md3: false,
+    noGuidelines: false,
+  },
+  {
+    name: 'Transfer List',
+    srcLight: '/static/material-ui/react-components/transfer-list-light.png',
+    srcDark: '/static/material-ui/react-components/transfer-list-dark.png',
+    link: '/material-ui/react-transfer-list/',
+    md1: false,
+    md2: false,
+    md3: false,
+    noGuidelines: true,
+  },
+  {
+    name: 'Toggle Button',
+    srcLight: '/static/material-ui/react-components/toggle-button-light.png',
+    srcDark: '/static/material-ui/react-components/toggle-button-dark.png',
+    link: '/material-ui/react-toggle-button/',
+    md1: false,
+    md2: false,
+    md3: false,
+    noGuidelines: true,
+  },
+];
+
+export default function MaterialInputComponents() {
+  return (
+    <Grid container spacing={2} sx={{ pt: 1 }}>
+      {inputComponents.map(({ name, link, srcLight, srcDark, md1, md2, md3, noGuidelines }) => (
+        <Grid item xs={12} sm={4} sx={{ flexGrow: 1 }} key={name}>
+          <ComponentShowcaseCard
+            link={link}
+            name={name}
+            srcLight={srcLight}
+            srcDark={srcDark}
+            md1={md1}
+            md2={md2}
+            md3={md3}
+            noGuidelines={noGuidelines}
+          />
+        </Grid>
+      ))}
+    </Grid>
```

Comment:
Do you think we should abstract this layout structure? It seems to be shared between most component sections. I don't have a strong opinion, and not abstracting is better than abstracting too early 😅, but if these sections should always look the same layout-wise, then maybe we should

Reply:
I'm open to suggestions if you have a specific one for abstracting! I felt like this whole setup was a bit repetitive, but also didn't want to go too deep into the rabbit hole. Let me know!

Reply:
Not going too deep into the rabbit hole seems like a good idea 👍🏼 let's leave it as is.